File: CSharp/SequenciaDeFibonacci/ClassPrint.cs
namespace SequenciaDeFibonacci
{
    public class ClassPrint
    {
        public static void Print(int minimo, int maximo)
        {
            int param1, param2 = 1, resultado = 1;

            for (int i = 0; i < maximo; i++)
            {
                param1 = param2;
                param2 = resultado;
                resultado = param1 + param2;

                if (i >= minimo && i <= maximo)
                {
                    Console.WriteLine($"{param1} + {param2} = {resultado}");
                }
            }
        }
    }
}

File: Python/SequenciaDeFibonacci/ClassPrint.py
def PrintFibonacci(minimo, maximo):
        
    param1, param2, resultado = 1, 1, 1

    for x in range(maximo):
    
        param1 = param2
        param2 = resultado
        resultado = param1 + param2
    
        if minimo <= x <= maximo:
            print(f"{param1} + {param2} = {resultado}")